src/commands/publish/preview/mod.rs

```diff
@@ -90,23 +56,37 @@ pub fn preview(
         };
 
         open_browser(&format!(
-            "https://cloudflareworkers.com/ui/staging/index.html?wrangler_session_id={}\\&wrangler_ws_port={}\\&hide_editor#{}:{}{}",
+            "https://cloudflareworkers.com/?wrangler_session_id={}\\&wrangler_ws_port={}\\&hide_editor#{}:{}{}",
             &session.to_string(), ws_port, script_id, https_str, preview_host,
         ))?;
 
+        //don't do initial GET + POST with livereload as the expected behavior is unclear.
```

Comment:
what do you mean here?

Reply:
we ignore the `wrangler preview get --watch` and `wrangler preview post --watch` commands as it is unclear what they should do. They will be addressed once we have a command for browserless preview